### PR TITLE
Update server APIs

### PR DESCRIPTION
**Pull request by chetanyakan**

### CODE REVIEW

File: server/token.go [hunk 1]
```diff
@@ -1,91 +1,85 @@
...
 
-func getEncryptionPassword(p *Plugin) []byte {
+func (p *Plugin) getEncryptionPassword() []byte {
 	//if the fallbackPassword is set this means the plugin cannot read from KV pair
 	if fallbackPassword != "" {
 		return []byte(fallbackPassword)
 	}
 
-	tokenSignPasswordByte, _ := p.API.KVGet(kvEncryptionPassword)
+	tokenSignPasswordByte, _ := p.API.KVGet(kvEncryptionPasswordKey)
```
Comment (hrishavkmr) [inline server/token.go:38]: Should we use KVLoad() here instead?

File: server/api.go [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+// body contains an array with file ids in JSON format
+func (p *Plugin) parseFileIDs(w http.ResponseWriter, r *http.Request) {
+	//extract fileIDs array from body
+	body, bodyReadError := ioutil.ReadAll(r.Body)
+	if bodyReadError != nil {
+		p.API.LogError("Error when reading body: ", bodyReadError.Error())
+		return
+	}
+	var fileIDs []string
+	_ = json.Unmarshal(body, &fileIDs)
```
Comment (manojmalik20) [inline server/api.go:96]: Shouldn't we handle the error while unmarshaling?

File: server/api.go [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+			}
+			files = append(files, file)
+		}
+	}
+
+	responseJSON, _ := json.Marshal(files)
+
+	w.Header().Set("Content-Type", "application/json")
+	if _, err := w.Write(responseJSON); err != nil {
+		p.API.LogError("failed to write status", "err", err.Error())
```
Comment (manojmalik20) [inline server/api.go:121]: I don't think "failed to write status" is the right error message. It should be like "failed to write response" or something else.

**Review by hrishavkmr — APPROVED** [review]

File: server/api.go [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+	}
+}
+
+// returnWopiFileList returns the list with file extensions and actions associated with these files
+func (p *Plugin) returnWopiFileList(w http.ResponseWriter, _ *http.Request) {
+	responseJSON, _ := json.Marshal(WopiFiles)
+	if _, err := w.Write(responseJSON); err != nil {
+		p.API.LogError("failed to write status", "err", err.Error())
```
Comment (manojmalik20) [inline server/api.go:129]: Same here

**Review by manojmalik20 — APPROVED** [review]

File: server/token.go (hunk 1 above)
Comment (chetanyakan) [reply server/token.go:38]: We don't need that here.

**Pull request merged**